sounds/models.py

```diff
@@ -456,6 +456,18 @@ def bulk_sounds_for_pack(self, pack_id, limit=None, include_analyzers_output=Fal
             qs = qs[:limit]
         return qs
 
+    def bulk_sounds_for_collection(self, collection_id, limit=None, include_analyzers_output=False):
+        qs = self.bulk_query(include_analyzers_output=include_analyzers_output)
+        qs = qs.filter(
+            moderation_state="OK",
+            processing_state="OK",
+            collections=collection_id
```

Comment:
Does this work?? I guess it is some feature of the `through` model, but I'm a bit surprised here... I'd like to look into this with @alastair